index.md

```diff
@@ -170,7 +170,7 @@ Processing actions often include, but are not necessarily limited to:
     - Retain original order or “as received” order whenever feasible.
     - Do not arrange material within a folder without the prior approval of the Assistant Director for Processing. 
 - Description
-    - Create, or revise, standard finding aid to file level, in
+    - Create, or revise, finding aid to file level, in
```

Comment:
seems like a lot of commas here. Maybe change to "Create or revise finding aid to file level in accordance with all RAC descriptive standards"? Not sure I'm reading this right. "to file level" is throwing me off a bit